core/src/plugins/filed/python/test/bareosfd_test.py

```diff
@@ -139,7 +142,7 @@ def test_RestorePacket(self):
     def test_IoPacket(self):
         test_IoPacket = bareosfd.IoPacket()
         self.assertEqual(
-            'IoPacket(func=0, count=0, flags=0, mode=0000, buf="", fname="<NULL>", status=0, io_errno=0, lerror=0, whence=0, offset=0, win32=0)',
+            'IoPacket(func=0, count=0, flags=0, mode=0000, buf="", fname="<NULL>", status=0, io_errno=0, lerror=0, whence=0, offset=0, win32=0, filedes=-1)',
```

Comment:
bareos-check-sources says that black wants to reformat this file

Reply:
done